src/app/containers/MostRead/index.test.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { act } from 'react-dom/test-utils';
+import MostReadContainer from '.';
+import { RequestContextProvider } from '#contexts/RequestContext';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import newsMostReadData from '#data/news/mostRead';
+
+let container;
+
+const mostReadContainerWithContext = (service, variant = null) => (
+  <RequestContextProvider
+    bbcOrigin={`http://localhost:7080/${service}/articles/c0000000000o`}
+    id="c0000000000o"
+    isAmp={false}
+    pageType="article"
+    service={service}
+    statusCode={200}
+    pathname={`/${service}`}
+    variant={variant}
+  >
+    <ServiceContextProvider service={service}>
+      <MostReadContainer />,
+    </ServiceContextProvider>
+  </RequestContextProvider>
+);
+
+describe('MostReadContainer', () => {
+  beforeEach(() => {
```

Comment:
I don think we need a `beforeEach` and `afterEach`.
We can just have a `beforeAll` to handle 
```
await act(async () => {
      fetch.mockResponse(JSON.stringify(newsMostReadData));
      ReactDOM.render(mostReadContainerWithContext('news'), container);
    }); 
```